vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -82,7 +85,17 @@ func main() {
 	metrics_quality.Register()
 
 	useCheckpoints := *storage != "prometheus"
-	recommender := routines.NewRecommender(config, *checkpointsGCInterval, useCheckpoints, *vpaObjectNamespace, *recommenderName)
+
+	var postProcessorsNames []routines.KnownPostProcessors
+	if *postProcessorCapping {
+		postProcessorsNames = append(postProcessorsNames, routines.Capping)
```

Comment:
Since we will have a bool flag for each post processor I think we can have a function that returns a list of post processors based on flags instead of the factory

Reply:
Does that mean that we're fixing the order in which the post-processors are applied in this place? This can make a difference, something like the capping post-processor should always be applied last, I guess?

Reply:
Yes. I think that we should fix order of postprocessors unless we have a good reason to allow changing it. I expect it would be easy to make mistake in order of postprocessors but hard to make things better by changing order from what we come up with.